ui/src/config.ts

```diff
@@ -5,8 +5,19 @@
 
 const credentials = require(process.env.REACT_APP_CONFIG_FILE as string);
 
-export const credentialsMap = credentials;
+export const preloadedCredentialsMap = credentials;
 
 export const isLocal = process.env.REACT_APP_IS_LOCAL === "true";
 
-export const httpBaseUrl = process.env.REACT_APP_HTTP_BASE_URL as string;
```

Comment:
You cannot **pre**configure the ledgerId / httpBaseUrl if you try to create a dit file. You just don't know it. That's why I included this from the DA Marketplace app.

I am open to suggestions.

Reply:
Let's use `getLedgerId` and throw exception when REACT_APP_LEDGER_ID is null.
REACT_APP_LEDGER_ID should be defined (it is not yet).